scripts/2.mutation-classifier.py

```diff
@@ -36,8 +38,28 @@
 
 # In[3]:
 
-# We're going to be building a 'TP53' classifier 
-mutation_id = '7157' # TP53
+
+# We're going to be building a classifier with multiple genes filtered by two diseases 
+# Example:
+# gene_ids = ['7157', '7158', '7159', '7161']
+
+# Information passed into the notebook is stored in environment variables
+gene_ids = os.environ.get('gene_ids')
+if not gene_ids:
+    gene_ids = ['7157'] # TP53 is the default
+else:
+    gene_ids = gene_ids.split('-')
+    
+disease_acronyms = os.environ.get('disease_acronyms')
+
+if not disease_acronyms:
+    disease_acronyms = [] # use all of the diseases as default
```

Comment:
I think it makes sense to keep `disease_acronyms = None` for the case where all diseases are used...